packages/editor/src/client/panel/articleMetadataPanel.tsx

```diff
@@ -137,6 +143,23 @@ export function ArticleMetadataPanel({
     }
   }, [data?.authors, authors, socialMediaAuthors])
 
+  const [createAuthor] = useCreateAuthorMutation({
+    refetchQueries: [getOperationNameFromDocument(AuthorListDocument)]
+  })
+  const [authorName, setAuthorName] = useState('')
+  const [authorSlug, setAuthorSlug] = useState('')
```

Comment:
i don't think you need a state for the slug. You can create the slug in `handleCreateAuthor()`